go/fn/result.go

```diff
@@ -279,18 +280,9 @@ func ErrorConfigObjectResult(err error, obj *KubeObject) *Result {
 
 func ConfigObjectResult(msg string, obj *KubeObject, severity Severity) *Result {
 	return &Result{
-		Message:  msg,
-		Severity: severity,
-		ResourceRef: &yaml.ResourceIdentifier{
-			TypeMeta: yaml.TypeMeta{
-				APIVersion: obj.GetAPIVersion(),
-				Kind:       obj.GetKind(),
-			},
-			NameMeta: yaml.NameMeta{
-				Name:      obj.GetName(),
-				Namespace: obj.GetNamespace(),
-			},
-		},
+		Message:     msg,
+		Severity:    severity,
+		ResourceRef: obj.GetId(),
```

Comment:
This `GetId` only does not return the `version` for `ResourceIdentifier`. Do I need to write a new function that fills every field in `ResourceIdentifier`, or this is enough?